src/app/public/modules/search/search.component.html

```diff
@@ -46,19 +51,16 @@
               [disabled]="disabled"
               (click)="clearSearchText()"
             >
-              <sky-icon
-                class="sky-search-icon-default"
+              <sky-icon *skyThemeIf="'default'"
                 icon="times"
-              >
-              </sky-icon>
-              <sky-icon
-                class="sky-search-icon-modern"
+              ></sky-icon>
+              <sky-icon *skyThemeIf="'modern'"
```

Comment:
The "X" button looks a little large in modern theme, when compared to the design specs. Saw this in Chrome/Mac.